NuRadioMC/SignalGen/emitter.py

```diff
@@ -75,12 +75,14 @@ def get_time_trace(amplitude, N, dt, model, full_output=False, **kwargs):
         sigma = half_width / (np.sqrt(2 * np.log(2)))
         trace = 1 / (sigma * np.sqrt(2 * np.pi)) * np.exp(-1 / 2 * ((time - 500) / sigma) ** 2)
         trace = amplitude * 1 / np.max(np.abs(trace)) * trace
-    elif(model == 'idl1' or model =='hvsp1'):  # the idl & hvsp waveforms gemerated in KU Lab stored in hdf5 file
+    elif(model == 'idl1' or model == 'hvsp1' or model == 'calPulser'):  # the idl & hvsp2 waveforms gemerated in KU Lab stored in hdf5 file
         path = os.path.dirname(os.path.dirname(__file__))
         if(model == 'idl1'):
-            input_file = os.path.join(path, 'data/idl1_data.hdf5')
-        else:
+            input_file = os.path.join(path, 'data/idl_data.hdf5')
```

Comment:
Was this intentional to change the file name, but not add a new file?

Reply:
This shouldn't work now.

Reply:
This is a typo, Thanks Anna.